tests/test_clock.py

```diff
@@ -26,9 +26,8 @@ def test_scheduler_has_job(self):
     def test_clock_expects_config_to_be_ready(self):
         assert not get_config().ready
         jobs = self.clock.scheduler.get_jobs()
-        with raises(RuntimeError) as excinfo:
+        with raises(RuntimeError, message='Config loading not finished'):
```

Comment:
@MatthewWilkes Except... it doesn't work the way I thought/hoped. It's just a way to set an alternative exception message of course. <face palm>